pkg/process/config/config.go

```diff
@@ -262,37 +262,22 @@ func getContainerHostType() model.ContainerHostType {
 	return model.ContainerHostType_notSpecified
 }
 
+// loadEnvVariable reads env variables specific to process-agent and overrides the corresponding settings
+// in the global Config object.
+// This function is used to handle historic process-agent env vars. New settings should be
+// handled directly in the /pkg/config/process.go file
 func loadEnvVariables() {
 	// The following environment variables will be loaded in the order listed, meaning variables
 	// further down the list may override prior variables.
 	for _, variable := range []struct{ env, cfg string }{
-		{"DD_PROCESS_AGENT_CONTAINER_SOURCE", "process_config.container_source"},
-		{"DD_SCRUB_ARGS", "process_config.scrub_args"},
-		{"DD_STRIP_PROCESS_ARGS", "process_config.strip_proc_arguments"},
 		{"DD_ORCHESTRATOR_URL", "orchestrator_explorer.orchestrator_dd_url"},
-		{"DD_HOSTNAME", "hostname"},
-		{"DD_BIND_HOST", "bind_host"},
 		{"HTTPS_PROXY", "proxy.https"},
-		{"DD_PROXY_HTTPS", "proxy.https"},
 	} {
```

Comment:
I opted to keep `HTTPS_PROXY` to not risk breaking it for customers that rely on it.

Reply:
That's fine I think. The core agent also handles `HTTPS_PROXY` so we should keep it